packages/web-app/src/components/appli/EntitiesForm/Organization/transformers.js

```diff
@@ -10,7 +10,7 @@ export const makePostOrganizationData = data => ({
     text: data.organization.name,
     language: data.organization.language
   },
-  postalCode: data.organization.zipCode,
+  postalCode: parseInt(data.organization.zipCode, 10),
```

Comment:
Is postal Code always an integer ? I am thinking of `01000` for "Bourg en Bresse" for example...

Reply:
Il y a des pays ou le code postal contient des lettres, ou il fait plus de 5 caractères. Un exemple en Angleterre EC4N 1SA